meteor/client/ui/Settings/helpers/config-manifest-table-entry-selector.ts

```diff
@@ -32,7 +32,12 @@ class ConfigManifestTableEntrySelector {
 
 		const options = sourceTable.flatMap((sourceRow) => this.getOptionsFromSourceRow(sourceRow, targetRow, item))
 		options.push({ value: DEFAULT_VALUE_FOR_NO_AVAILABLE_OPTION, label: DEFAULT_VALUE_FOR_NO_AVAILABLE_OPTION })
-		return options
+		return this.removeDuplicateOptions(options)
+	}
+
+	private removeDuplicateOptions(options: SelectOption[]): SelectOption[] {
+		const map: Map<string, SelectOption> = new Map(options.map((option) => [option.value, option]))
+		return [...map.values()]
```

Comment:
Can't this be reduced to a one-liner?
```suggestion
		return [...new Map(options.map((option) => [option.value, option]).values()]
```

Reply:
Did consider that syntax. Personally I think too much is happening on one line and I like that we explicit can see that we map it into a Map. 
In the end, this is a personal preference thing.